src/middlewared/middlewared/plugins/disk.py

```diff
@@ -280,6 +280,11 @@ async def do_update(self, id, data):
             else:
                 await self.middleware.call('disk.toggle_smart_off', new['name'])
 
+            invalid_smart_flags = ['-a', '-d', '-n', '-W', '-m', '-M', 'exec']
```

Comment:
It's too late to validate at this point. The disk has already been updated at L263

Reply:
oof, thank you. good catch, will fix